js/views/attachment_view.js

```diff
@@ -198,7 +206,7 @@
         if (!this.objectUrl) {
             this.objectUrl = window.URL.createObjectURL(this.blob);
         }
-        this.view = new View(this.objectUrl, this.model.contentType);
+        this.view = new View(this.objectUrl, this.model.contentType, this.blob);
```

Comment:
What if the third parameter here was an options hash? I generally like functions with 1 or 2 parameters, the second as an options hash.

Reply:
Great idea! Not a fan of too many positional arguments myself 😄